warehouse/pom.xml

```diff
@@ -444,6 +444,19 @@
             <groupId>gov.nsa.datawave</groupId>
             <artifactId>datawave-common-test</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+        </dependency>
```

Comment:
Are these needed here?  and if so, shouldn't they be test scoped?

Reply:
Yes these are needed and yes they should be test scoped.

Reply:
the warehouse module itself has no source code, which means it has no tests to compile, let alone run, so why do you think these are needed?  for what it's worth I removed all of the dependencies in this pom and everything built just fine...

Reply:
happy to approve once this is removed.